nixos/modules/services/security/keylime-agent.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.keylime-agent;
+in
+{
+  options = {
+    services.keylime-agent = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = lib.mdDoc "Whether to enable keylime-agent service.";
+      };
+
+      stateDir = mkOption {
+        type = types.path;
+        default = "/var/lib/keylime/";
+        defaultText = literalExpression "/var/lib/keylime/";
+        description = lib.mdDoc "State directory for keylime";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "keylime";
+        description = lib.mdDoc "User which runs keylime.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "tss";
+        description = lib.mdDoc "Group which runs keylime.";
+      };
```

Comment:
I know this tends to break some services, but have you experimented with running this as `DynamicUser`? It doesn't seem to have any socket stuff going on so it would be great if this is possible.